h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -845,4 +872,9 @@ public static Frame iris() {
       return parse_test_file(Key.make("iris.hex"), "smalldata/iris/iris_wheader.csv");
     }
   }
+
+  public interface ParseSetupTransformer {
```

Comment:
At least basic JavaDoc with explanation would help others understand purpose of this interaface and its method, even if it can be red from the context.